### ISSUE TITLE
fromURL, loadBuffer, decodeStream missing

### ISSUE THREAD

**Issue by edemaine** [cheeriojs/cheerio#3101]
The [documentation](https://cheerio.js.org/docs/api#fromurl) says that this should work:

```ts
import * as cheerio from 'cheerio';
cheerio.fromURL('https://example.com')
```
or
```ts
require('cheerio').fromURL('https://example.com')
```

However, `cheerio` does not appear to have a `fromURL` property, in the released NPM version 1.0.0-rc.12. Perhaps related to #2774? Similarly, `decodeStream` which `fromURL` is built on, `loadBuffer`, `stringStream` all seem to be missing. I have `contains`, `load`, `html`, `merge`, `parseHTML`, `root`, `text`, `xml`.

As a first-time user to Cheerio, this was pretty confusing, as it seems like the easiest way to get started with the library.

**Comment by AlGhaliyahM** [comment]
@edemaine I have faced the same issue and revert to cheerio documentation. As per the documentation 
`Only the load method is available by default. If you are using current NodeJS version, other methods will be available if you use an ES Module imports (your file has a .mjs extension, or you have type: "module" in your package.json). You can also directly import or require cheerio/lib/batteries.`

I have directly imported cheerio/lib/batteries to use fromUrl property but it seems to be missing.

**Comment by edemaine** [comment]
FWIW, I'm already using ESM via `type: "module"`. The word `fromURL` does not appear in any files within `node_modules/cheerio`, so my guess is that it's an issue of code not yet published to NPM.

**Comment by fb55** [comment]
This hasn't been released yet — as the banner at the top of the website says:

> Cheerio's website is still a work-in-progress, and covers Cheerio's next release that isn't available yet.

I want to wrap up work on the website before issuing the release, but haven't been able to focus on that for a while.

**Comment by edemaine** [comment]
Thanks for confirming! I'm new to Cheerio so it was just a confusing on-boarding experience. (I just wrote my own fromURL for now.)

The site looks great! Let me know if there's something you could use help with. I did have a wording suggestion for newcomers, so I'll go make a PR.

**Comment by michaelfaith** [comment]
Would love for this to become GA.  It's such a great option for us

**Comment by realChakrawarti** [comment]
Coming to this after 4 months, 
Any update on this? Wasted quite a few minutes to figure out what's wrong with my setup. 

Please flag the docs accordingly: https://cheerio.js.org/docs/api#fromurl. Will raise a PR if required.

**Comment by laygir** [comment]
I do not see any implementation of `fromURL` in the package I get from npm. And unable to `require` from "cheerio/lib/batteries" as it does not appear to exist. What am I missing?

**Comment by michaelfaith** [comment]
As they said, it hasn't been released yet

**Comment by krel99** [comment]
It does appear that new loading documents have still not been released. I am upvoting this as it is confusing to work with a documentation of "future" when building tools for today.

**Comment by fb55** [comment]
Fixed with the [1.0 release](https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0)!
